### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by yeetivity**
Saving results and displaying them for everywhere in the app, working. 
Fixed some bugs as well

### CODE REVIEW

File: app/src/main/java/kth/jjve/xfran/HomeScreenActivity.java [hunk 1]
```diff
@@ -37,7 +37,8 @@ protected void onCreate(Bundle savedInstanceState) {
         WODDescription = findViewById(R.id.home_WODDescription);
         WODExercises = findViewById(R.id.home_WODExercises);
         Username = findViewById(R.id.home_username);
-        PlannedWorkouts = findViewById(R.id.home_rv);
+        //Todo: get the planned workouts of the day to the home view
+//        PlannedWorkouts = findViewById(R.id.home_rv);
```
Comment (elisakaisa) [inline app/src/main/java/kth/jjve/xfran/HomeScreenActivity.java:41]: is the todo here normal? or is that the recycler view that we took away in the home screen)

File: app/src/main/java/kth/jjve/xfran/MontlyCalendarActivity.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import androidx.recyclerview.widget.RecyclerView;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import kth.jjve.xfran.adapters.MonthlyCalendarAdapter;
+import kth.jjve.xfran.utils.CalendarUtils;
+import kth.jjve.xfran.viewmodels.CalendarVM;
+
+public class MontlyCalendarActivity extends BaseActivity implements MonthlyCalendarAdapter.OnItemListener {
```
Comment (elisakaisa) [inline app/src/main/java/kth/jjve/xfran/MontlyCalendarActivity.java]: typo in the class name -> MonthlyCalendarActivity

File: app/src/main/java/kth/jjve/xfran/viewholders/MontlyCalendarViewHolder.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+import kth.jjve.xfran.R;
+import kth.jjve.xfran.adapters.MonthlyCalendarAdapter;
+
+public class MontlyCalendarViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
```
Comment (elisakaisa) [inline app/src/main/java/kth/jjve/xfran/viewholders/MontlyCalendarViewHolder.java]: refactor to MonthlyCalendarViewHolder

**Review by elisakaisa — APPROVED** [review]
refacror the typo in monthly in the 2 classes, but otherwise awesome job!

File: app/src/main/res/layout/act_result_save.xml [hunk 4]
```diff
@@ -155,6 +155,7 @@
                 android:importantForAutofill="no"
                 android:minHeight="48dp"
                 android:textAlignment="textEnd"
+                android:inputType="none"
```
Comment (MariahSabioni) [inline app/src/main/res/layout/act_result_save.xml:159]: Add a to-do to correct the keyboard bug (right now does not display a : and we set that on the input digits).

**Review by MariahSabioni — APPROVED** [review]
Would only add the todo as a reminder - nothing major tho.
Another todo that I wanted to add as a comment but could not (because it was not changed in this pull request) is on EventRepo line 120, to see if we can change the Event construction to the toObject method.
Otherwise looks nice and ready to present, great job ^^

**Pull request merged**